index.js

```diff
@@ -16,6 +16,9 @@ const CONNECTIONS_COUNT = parseInt(process.env.CONNECTIONS_COUNT || "1")
 const MAX_CONNECTION_CONCURRENCY = parseInt(process.env.MAX_CONNECTION_CONCURRENCY || "10")
 const XRP_NODE_URL = process.env.XRP_NODE_URL || 'wss://s2.ripple.com'
 const EXPORT_TIMEOUT_MLS = parseInt(process.env.EXPORT_TIMEOUT_MLS || 1000 * 60 * 5)     // 5 minutes
+// Although we request the last 'validated' block we are seeing blocks which are neither validated nor even closed. We introduce this extra delay to prevent this.
+// In terms of time, this is a delay between 1 and 2 minutes.
+const EXTRA_CONFIRMATIONS_BLOCKS = parseInt(process.env.CONFIRMATIONS || "20")
```

Comment:
In other projects, for example [erc20-transfers-exporter](https://github.com/santiment/erc20-transfers-exporter/blob/2c50157648b6485bacd088678bfe301ed7ed418f/index.js#L37)
, we already have similar parameter and we name it just `CONFIRMATIONS`, maybe for consistency use same name in this project too?

Reply:
Yes that makes sense. Changed.